src/main/java/microsoft/exchange/webservices/data/property/complex/AttachmentCollection.java

```diff
@@ -107,7 +108,8 @@ public FileAttachment addFileAttachment(String fileName) {
    * @return A FileAttachment instance.
    */
   public FileAttachment addFileAttachment(String name, String fileName) {
-    FileAttachment fileAttachment = new FileAttachment(this.owner);
+    FileAttachment fileAttachment = new FileAttachment(
+        this.owner, this.owner.getService().isTraceEnabledFor(TraceFlags.EwsResponse));
```

Comment:
Using the "trace enabled" is a great idea.

It also extends our hackery of their library.